app.py

```diff
@@ -331,6 +372,11 @@ def open_file_dialog(self, error_label):
         except FileNotFoundError as error:
             logging.info(error)
             return
+        except Exception as error:
```

Comment:
I used such exceptions intentionally to avoid writing code that handles different errors, each in a different way.
I used it only in places where the file is read and in my tests it proved to be an efficient solution to the problem.

Reply:
I am not entirely convinced, but we can leave it as it is